data/intro missions.txt

```diff
@@ -61,7 +61,7 @@ mission "Intro [0]"
 			`	"Great," he says. "My name is James, by the way."`
 			`	"<first> <last>," you say.`
 			`	"I'm looking forward to traveling with you, Captain <last>," says James.`
-			`	As you're helping him wheel his luggage aboard and showing him his bunk, he says, "Before we take off, you might head on over to the trading center and stock up on medical goods; we can sell those for a good profit on <planet>. Or take a look at the outfitter if you like, but they don't sell much here compared to on the big manufacturing worlds."`
+			`	As you're helping him wheel his luggage aboard and showing him his bunk, he says, "Before we take off, you might head on over to the trading center and stock up on medical goods; we can sell those for a good profit on <planet>.  Also make sure to visit the local outfitter, while they don't sell much here compared to on the big manufacturing worlds, every world with an outfitter will at least carry a map with information on the neighboring systems, I'd suggest buying one as it takes a bit of the guesswork out of navigation. It isn't a requirement, though: Some captains like exploring instead."`
```

Comment:
Sweet run-on.
```suggestion
			`	As you're helping him wheel his luggage aboard and showing him his bunk, he says, "Before we take off, you might head on over to the trading center and stock up on medical goods; we can sell those for a good profit on <planet>. Also make sure to visit the local outfitter. While they don't sell much here compared to on the big manufacturing worlds, every world with an outfitter will at least carry a map with information on the neighboring systems. Buying them can help if you're not quite sure where you're going."`
```
I'd rather not word it as, "I'd suggest you buy one, but you don't have to it you don't want to," as that seems very ham-fisted to me. Let's instead just describe the function of the map, how it can help, and leave it to the player as to whether or not they want to buy one.